UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.sln.DotSettings

```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=catalogue/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=UKHO/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
Can we remove this file from the branch please as it's not something we normally have checked in.

Reply:
Yes removed from the branch. Thanks!